### ISSUE TITLE
in gbdt, Tree0's leaning rate = 1 and others is equal to config ${learning rate}

### ISSUE THREAD

**Issue by hlee13** [Microsoft/LightGBM#1889]
Tree=0
num_leaves=63
num_cat=0
split_feature=25 26 27 26 22 24 25 27 5 25 25 25 27 27 24 25 25 25 3 22 22 5 0 5 26 3 9 9 5 22 12 5 22 3 15 9 23 19 25 27 10 27 6 24 27 22 18 21 27 5 21 19 0 24 25 25 9 7 20 13 21 25
split_gain=264.633 147.999 163.201 66.7994 64.9188 80.766 52.2226 38.088 32.4414 30.2777 41.2374 29.9636 27.0313 23.3768 25.1322 22.04 22.2517 20.4465 16.3604 15.0237 14.2878 13.6828 13.2111 13.1653 12.4482 12.354 11.9956 11.7455 11.556 10.6827 10.3965 9.93505 8.92007 8.77787 8.6175 8.41735 9.77324 8.16374 8.08136 10.7352 9.12798 7.86836 7.63384 7.4482 9.61124 7.29484 7.26322 7.03878 7.03012 11.6878 6.78634 6.49801 6.36233 6.23677 5.93125 10.4457 5.89306 5.7646 6.6416 5.286 4.99656 5.8532
threshold=1.2815000000000003 0.77250000000000008 0.90450000000000019 0.87350000000000005 1.0485000000000002 1.0265000000000002 0.94550000000000012 0.84050000000000014 0.70050000000000001 0.67850000000000021 1.0025000000000002 0.52350000000000019 0.72650000000000003 0.89550000000000007 1.0845000000000002 0.62350000000000005 0.98950000000000016 0.65450000000000019 1.5555000000000003 1.0285000000000002 0.94650000000000001 0.88650000000000018 1.2935000000000001 1.3365000000000002 0.97650000000000003 1.8335000000000001 0.83350000000000002 1.1155000000000002 0.48450000000000004 0.72250000000000003 1.0000000180025095e-35 0.77950000000000019 0.87650000000000017 1.3585000000000003 -0.11549999999999999 0.70950000000000013 1.0025000000000002 -0.28149999999999992 1.7500000000000002 1.413 -0.17249999999999996 0.83650000000000013 -1.2974999999999997 1.2635000000000003 1.0145000000000002 1.2515000000000003 -1.0374999999999999 0.88850000000000018 0.85750000000000004 0.88150000000000006 0.83350000000000002 -0.18749999999999997 0.71650000000000003 0.72150000000000014 0.98450000000000004 0.86450000000000016 0.87450000000000017 0.65550000000000008 1.0000000180025095e-35 0.83250000000000013 0.95150000000000012 0.80750000000000022
decision_type=2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2
left_child=1 8 3 6 5 22 7 11 29 26 28 -3 23 20 19 -7 18 51 60 -15 32 -8 37 30 41 38 -5 35 -11 -1 52 57 -2 -29 -21 45 54 -4 40 -40 -16 -12 -31 -18 -45 -19 -44 -39 49 -14 -22 -6 -10 -48 55 -37 -26 58 59 -13 61 -17
right_child=13 2 4 9 17 15 21 -9 12 10 24 31 48 14 25 16 43 27 -20 34 50 -23 -24 -25 56 -27 -28 33 -30 42 -32 -33 -34 -35 -36 36 -38 47 39 -41 -42 -43 46 44 -46 -47 53 -49 -50 -51 -52 -53 -54 -55 -56 -57 -58 -59 -60 -61 -62 -63
leaf_value=0.12176214176072674 0.083311602501405069 0.095042103949563486 -0.015818787622058766 0.095345740742872151 0.13796420105167756 0.043011361367877213 0.02272302472429566 0.05733189537571555 0.11901758061601457 0.17811713059881651 0.17166686079204721 0.18555285829273113 0.0070953936387802008 -0.049254880089801722 0.14387819842683755 0.18689372722114198 0.027239455936839796 0.24856086664642874 0.10089995839278004 -0.023646346230090737 0.18555285829273113 0.14528769134643876 -0.061462015699112665 0.24110211812374038 0.17544026862900725 -0.036536895282115095 0.20553077727758612 0.23297085948835064 0.27691976124413703 0.10316499544415711 0.079703087591122962 0.25068266592497129 -0.023864272577700713 0.14624972619632529 0.071965707658104311 0.073764884318116397 0.20005862215463016 0.0089192513750234043 -0.0016150302312341047 0.1111955813535849 0.050724301915122602 0.071042651806445922 -0.043491934114901293 0.18347085453843504 0.06878051493055079 0.14528769134643876 0.066062443490521733 0.10800884091333578 -0.0092632072878320038 0.13156300938474236 0.093845550067784111 0.048098120636652239 0.1978894065121804 0.0092689140416167548 0.051709759802267913 0.19515170677033 0.26969398746071238 0.24212904726816847 0.22412569570491325 0.095134409534729319 0.16740968271957946 0.27180729954214078
leaf_count=57 72 87 98 76 60 106 118 82 77 51 83 54 54 249 86 61 55 57 78 67 54 53 100 51 50 53 83 183 699 50 102 249 55 63 87 59 61 53 105 50 83 50 61 50 71 53 129 63 55 69 81 70 88 195 54 55 57 69 64 50 75 70
internal_value=0 0.217021 0.406973 0.791993 -0.0697758 -0.434173 0.357351 0.614791 -0.469346 1.10445 1.23481 0.797585 -0.10771 -0.902297 -1.07625 0.0179827 0.207798 0.378713 0.554741 -1.42414 -0.384435 -0.628747 -1.24921 0.242274 0.511854 -0.704361 0.292781 0.538451 1.46616 -0.833922 0.0640814 0.991459 -0.866888 0.871762 -0.932174 0.29658 0.0777409 -0.968238 -0.557651 -0.888101 -0.254576 0.102526 -0.940805 -0.352042 -0.0741241 0.752163 -1.03647 -0.608505 -0.732962 -0.466669 0.0694284 -0.340109 0.374969 -0.917045 -0.171702 0.0874307 1.02065 0.697794 0.497514 0.184968 0.850685 1.08682
internal_count=5600 4558 3570 1975 1595 880 826 655 988 1149 990 573 496 1042 780 566 460 715 284 403 262 171 314 318 240 377 159 585 750 492 267 486 127 246 154 339 229 214 324 155 169 133 435 176 121 110 385 116 178 123 135 130 165 324 168 114 107 237 168 104 206 131
shrinkage=1

Tree=1
num_leaves=63
num_cat=0
split_feature=25 26 25 25 3 5 22 26 5 5 24 26 24 13 9 26 9 22 25 14 11 22 25 22 3 19 5 11 26 9 13 22 26 25 3 9 18 5 22 23 13 11 9 9 26 3 3 11 19 15 24 20 18 5 22 11 3 26 24 13 17 3
split_gain=214.203 120.373 73.5775 92.5218 38.1803 26.7196 24.2361 22.3631 20.9675 19.0552 18.8009 18.4668 15.4984 15.104 14.1767 13.6348 13.0627 11.9664 11.6643 10.7936 10.7116 10.6581 11.6608 10.4615 11.618 9.35192 9.15311 8.95501 8.33363 8.0682 10.163 9.6062 7.91177 7.88038 7.48421 7.48181 6.90903 6.89789 6.85781 6.84198 6.61351 6.51248 6.41991 8.68628 8.97948 8.01413 6.11849 6.07324 8.8896 7.37318 6.04623 6.02489 5.97518 5.89833 5.6076 5.18958 5.04944 4.96599 4.95318 5.43839 5.29033 4.8161
threshold=1.2815000000000003 0.77250000000000008 0.62350000000000005 0.98450000000000004 1.6085000000000003 1.3365000000000002 1.0355000000000001 1.0345000000000002 0.88150000000000006 0.88150000000000006 0.72150000000000014 0.98250000000000004 1.2485000000000002 0.73250000000000004 1.0205000000000002 1.1705000000000003 0.85550000000000004 0.94650000000000001 0.76850000000000007 0.61450000000000016 -0.92749999999999988 0.96950000000000014 2.0390000000000001 1.0605000000000002 1.1495000000000002 0.78450000000000009 1.1815000000000002 0.5375000000000002 0.65150000000000008 1.0365000000000002 1.2425000000000002 0.77250000000000008 0.72850000000000004 0.7985000000000001 1.2105000000000004 1.5135000000000003 0.14600000000000002 0.82950000000000002 0.8135 1.0345000000000002 0.65150000000000008 -0.070499999999999993 0.64450000000000018 0.83350000000000002 0.85850000000000015 0.44850000000000007 0.79750000000000021 1.1820000000000002 -0.69249999999999978 -0.16949999999999998 0.75750000000000017 1.0000000180025095e-35 0.11700000000000001 0.73450000000000004 0.86050000000000015 -0.16349999999999998 1.6175000000000002 0.85650000000000015 1.0795000000000001 0.79250000000000009 0.9145000000000002 0.97650000000000003
decision_type=2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2
left_child=1 5 29 4 7 10 9 8 23 14 25 17 21 -6 52 26 47 -2 37 51 -14 38 39 24 42 32 27 -11 -27 30 31 -3 46 40 -22 58 -31 -9 -13 41 -10 -23 -4 44 -44 -45 53 48 -12 -50 -33 -8 -5 -1 -26 -48 -52 -35 59 -20 -60 -25
right_child=11 2 3 6 13 -7 19 18 33 15 16 12 20 -15 -16 -17 -18 -19 35 -21 34 22 -24 61 54 28 -28 -29 -30 36 -32 50 -34 57 -36 -37 -38 -39 -40 -41 -42 -43 43 45 -46 -47 55 -49 49 -51 56 -53 -54 -55 -56 -57 -58 -59 60 -61 -62 -63
leaf_value=-0.023934875818509792 -0.077050062268299302 0.012959976696260356 0.10957525673199085 -0.13461253971946324 0.070640285544173759 0.075820968499302838 0.083307626628124798 -0.06260347663587694 0.045854689596271508 0.037931937022054815 -0.059161189971844774 -0.073925538807816549 0.012236991508479331 -0.029104418701372054 0.0056539177930036086 -0.061885668947515043 -0.016113373604324639 0.024396609598686687 -0.028931076227009817 -0.022008322385067808 -0.041443625250426538 -0.12318651041451659 -0.16605994557786294 0.15158398626071717 0.046896167838315767 -0.12416667364755758 0.078724938956680451 -0.066332712981588757 -0.022609353342750774 -0.029940788877470832 0.032078889837404839 -0.016445708895612952 0.062164234093101602 0.09051996084926392 -0.11496862778116629 0.12697742587553437 0.048500906562375774 0.031150173897263458 -0.15499511686205811 -0.00026575463105936057 0.11814551493693855 -0.045510480575643254 -0.045766056042881434 0.013165605451572152 0.069690812429578752 0.1061649467306483 -0.0091742053757177637 -0.033836732911505456 -0.1730472402828182 -0.098163396361573202 -0.07206164265203667 0.011236483056820578 -0.063597741273297967 0.061151328549178811 -0.046938877531439861 -0.09952902373932454 -0.14202636665735893 0.15014322610861564 0.11718095564436032 0.055313162729966128 0.042295652606634412 0.074845145909078911
leaf_count=83 89 61 143 89 91 72 167 50 78 86 107 51 70 187 76 89 105 98 50 81 113 85 57 86 51 56 94 54 77 99 58 56 62 68 109 66 83 86 231 51 150 88 54 50 55 150 50 54 91 125 158 65 102 54 52 52 56 340 70 81 84 54
internal_value=0 0.196459 0.368645 0.494906 0.727356 -0.421832 -0.041396 0.833846 0.969162 -0.280344 -0.513768 -0.810058 -0.934575 0.0348848 -0.676192 0.0475657 -0.785785 -0.239803 0.465566 0.410576 -0.560294 -1.12856 -0.852678 0.727739 0.603858 -0.211273 0.302385 -0.0226319 -0.653634 -0.290272 -0.453019 -0.588487 -0.0158559 1.23381 -0.775627 0.65948 0.0583188 -0.0332932 -1.40334 -0.64695 0.934095 -0.836862 0.742933 0.57992 0.123203 0.82903 -0.181252 -0.95961 -1.06348 -1.29712 -0.750624 0.631096 -0.967087 0.095357 -0.00545699 -0.552521 -0.90387 1.40176 0.518288 0.231314 0.763062 1.21981
internal_count=5600 4558 3570 2999 2096 988 903 1818 1331 590 916 1042 855 278 267 323 482 187 487 313 292 563 281 695 555 434 234 140 133 571 389 331 301 636 222 351 182 136 282 224 228 173 452 309 109 200 239 377 323 216 270 232 191 137 103 102 214 408 285 131 154 140
shrinkage=0.1

**Comment by hlee13** [comment]
run examples/binary_classification/

**Comment by guolinke** [comment]
Refer to 
https://github.com/Microsoft/LightGBM/issues/1514
